tests/test_schema.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework_json_api.schemas.openapi import AutoSchema
+from tests.serializers import CallableDefaultSerializer
+
+
+def test_schema_callable_default():
```

Comment:
If the test is testing part of a class we also use a class to group the test.

So the test would look like this:

```
class TestAutoSchema:
    def test_map_serializer_with_callable_default(self):
        ...
```

Reply:
Punted into a class.